src/Controller/Component/PaginatorComponent.php

```diff
@@ -335,7 +336,10 @@ public function validateSort(RepositoryInterface $object, array $options)
             if (!in_array($direction, ['asc', 'desc'])) {
                 $direction = 'asc';
             }
-            $options['order'] = [$options['sort'] => $direction];
+
+            // Union the sort option from URL and default sort option to allow sorting by multiple fields
```

Comment:
I don't think so:  
Since the sort option from the URL always comes first, this is the column that will be sorted. The default sort order is only recognized if some rows of the data have the same value in the first (selected) sort column.

Imagine you have a list of users (first_name, last_name, any_score).  
The default sort option would be `['last_name' => 'ASC', 'first_name' => 'ASC']`.  
If you sort by the `any_score` column (e.g. with the PaginatorHelper), the result will be sorted by the `any_score` column first.  
However, if two users have the same `any_score`, what would you do in this case? Letting the database decide which user comes first? This might change, depending on the database engine.  
I would rather create a secondary sort by `last_name` and `first_name`, so the result is always the same, no matter what the default sort from the database would be.

And for completeness, if you really want to sort only on a single column, you could simply omit the `sort` key in the pagination settings.